ReactAndroid/src/main/java/com/facebook/react/modules/network/ForwardingCookieHandler.java

```diff
@@ -129,6 +129,7 @@ public void run() {
       for (String cookie : cookies) {
         addCookieAsync(url, cookie);
       }
+      getCookieManager().flush();
```

Comment:
It looks like the cookies are added asynchronously. Can we guarantee cookies have finished being added before they are flushed / written to permanent storage?

Reply:
Yes！

Reply:
![image](https://user-images.githubusercontent.com/8134572/46202329-22e10000-c349-11e8-98e8-d0840d1f2ea5.png)

Reply:
merge it,please